persistence/cosmosdb/saga-concurrency_pessimistic-locking_CosmosDB_[2,].partial.md

```diff
@@ -0,0 +1,35 @@
+## Sagas concurrency control
```

Comment:
This partial needs a rename to start with 1.1

Reply:
Renamed